test/dialect/postgresql/test_types.py

```diff
@@ -1420,9 +1421,11 @@ def test_array_agg_generic(self):
         argnames="with_enum, using_aggregate_order_by",
     )
     def test_array_agg_specific(self, with_enum, using_aggregate_order_by):
-        from sqlalchemy.dialects.postgresql import aggregate_order_by
-        from sqlalchemy.dialects.postgresql import array_agg
-        from sqlalchemy.dialects.postgresql import ENUM
+        from sqlalchemy.dialects.postgresql import (
+            ENUM,
+            aggregate_order_by,
+            array_agg,
```

Comment:
just fyi we use zimports that formats the imports as before

Reply:
I ran it and it left it that way. Both like `zimport <changed-file>` and with `pre-commit`.
Did I miss anything?

Reply:
Probably it does not support imports inside the functions, so it skips these.

Reply:
@CaselIT IIRC, zimports doesn't touch imports in functions, try/except blocks, or if/else statements. I think it just looks for "zero indented" imports.